packages/webapp/components/layouts/FooterNavBarLayout.tsx

```diff
@@ -2,22 +2,30 @@ import React, { ReactElement, ReactNode, useContext } from 'react';
 import dynamic from 'next/dynamic';
 import ProgressiveEnhancementContext from '@dailydotdev/shared/src/contexts/ProgressiveEnhancementContext';
 import useSidebarRendered from '@dailydotdev/shared/src/hooks/useSidebarRendered';
+import { Post } from '@dailydotdev/shared/src/graphql/posts';
 
 const FooterNavBar = dynamic(
   () => import(/* webpackChunkName: "footerNavBar" */ '../FooterNavBar'),
 );
 
-type FooterNavBarLayoutProps = { children?: ReactNode };
+interface FooterNavBarLayoutProps {
+  children?: ReactNode;
+  post?: Post;
+}
 
 export default function FooterNavBarLayout({
   children,
+  post,
 }: FooterNavBarLayoutProps): ReactElement {
   const { windowLoaded } = useContext(ProgressiveEnhancementContext);
   const { sidebarRendered } = useSidebarRendered();
 
   return (
     <>
-      <FooterNavBar showNav={sidebarRendered === false && windowLoaded} />
+      <FooterNavBar
+        showNav={sidebarRendered === false && windowLoaded}
+        post={post}
```

Comment:
NewComment button needs post data. So we need to inject them to footer navbar now.